#Request
geometry_msgs/Point[] position_list
geometry_msgs/Point[] orientation_list
---
#Response
bool success